todo/android/todo/src/main/java/com/mongodb/stitch/sdk/examples/todo/TodoListAdapter.java

```diff
@@ -24,7 +24,7 @@
 
 public class TodoListAdapter extends ArrayAdapter<TodoItem> {
 
-    private final MongoClient.Collection _itemSource;
+    private final RemoteMongoCollection _itemSource;
```

Comment:
[minor] Have you run this app? I'm not sure the updateOne here actually works since it would return a RemoteUpdateResult. Why the IDE doesn't complain about it? Beats me.

Reply:
Whoops. Fixed use of the RemoteMongoCollection methods, and confirmed that adding/removing/refreshing works